styles/asciidoc.atom-text-editor.less

```diff
@@ -87,6 +87,10 @@ atom-text-editor, :host {
         font-size: @font-size * .8;
         color: mix(green, @syntax-text-color, 50%);
       }
+
+      &.meta.attrlist {
```

Comment:
I have try and I'm ok with 'custom' styling.

Can you simple adding `&.meta.attrlist` to the existing  style section like that ?

```less

&.heading.blocktitle,
      &.substitution,
      &.meta.attrlist {
        color: @syntax-text-color-unobtrusive;
      }
```